assets/users/components/EditUser.jsx

```diff
@@ -69,15 +69,15 @@ function EditUser({user, onChange, errors, companies, onSave, onResetPassword, o
                         value={user.role}
                         onChange={onChange}
                         error={errors ? errors.role : null} />
-
+                    { currentUser.user_type === 'administrator' &&
```

Comment:
It would be good to have a utility function to determine if a user is an administrator, rather than checking against a string

Reply:
Added a function in utils